test/jdk/tools/jextract/testStruct/struct.h

```diff
@@ -51,6 +51,7 @@ struct AllTypes {
     unsigned long long ull;
     float f;
     double d;
+    long double ld;
```

Comment:
Should we add tests for when the unsupported types appear in different positions:

* return type
* argument type
* struct field
* global